src/Command/PluginListCommand.php

```diff
@@ -85,6 +88,9 @@ public function execute(Arguments $args, ConsoleIo $io): ?int
     public function buildOptionParser(ConsoleOptionParser $parser): ConsoleOptionParser
     {
         $parser->setDescription('Displays all currently available plugins.');
+        $parser->addOption('composer-path', [
```

Comment:
Is this option necessary? Isn't the `composer.lock` always going to be under the `ROOT` path?

Reply:
I primarily added this option to make testing easier.